earth_enterprise/integration_test/step_impl/genewimageryresource.py

```diff
@@ -0,0 +1,79 @@
+from getgauge.python import step, before_scenario, Messages
+import os, random, fnmatch, subprocess, datetime
+from subprocess import Popen
+import shutil
+
+# ------------------------
+# Helpful Not-Step Methods
+# ------------------------
+def get_env_value(sKey):
+   return os.getenv(sKey, "unset")
+
+def doesKhassetFileExist(sPathUntilDotKiasset):
+   return os.path.exists(sPathUntilDotKiasset + ".kiasset/khasset.xml" )
+
+def executeCommand(sTestDir):
```

Comment:
Consider re-using existing function [here](https://github.com/google/earthenterprise/blob/master/earth_enterprise/integration_test/step_impl/assets.py)instead of creating new one.

Reply:
Reused some portions.  There are more changes necessary to do a stronger integration, which we can avoid, until we have more time and priority to strengthen the open GEE test coverage and tests